modules/ui/src/main/scala/lucuma/ui/enums/ExecutionEnvironment.scala

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.ui.enums
+
+import cats.Eq
+import cats.derived.*
+import cats.syntax.all.*
+import eu.timepit.refined.collection.NonEmpty
+import eu.timepit.refined.types.string.NonEmptyString
+import lucuma.refined.*
+
+enum ExecutionEnvironment(val suffix: Option[NonEmptyString]) derives Eq:
```

Comment:
No Enumerated then?

Reply:
This looks like a purely runtime thing, I don't think we will ever put it in a database.